app/main/plugins/core/index.js

```diff
@@ -1,9 +1,11 @@
+/* eslint-disable import/no-mutable-exports, global-require */
+
 // Core plugins
 export { default as apps } from './apps'
 export { default as autocomplete } from './autocomplete'
-export { default as contacts } from './contacts'
+export let contacts
```

Comment:
I'd just rewrite this block to module.exports instead of static export. It will allow exporting some plugins conditionally, instead of exporting `undefined` and add conditions above.